subsys/mpsl/Kconfig

```diff
@@ -12,7 +12,7 @@ config MPSL_THREAD_COOP_PRIO
 	default SDC_RX_PRIO if BT_LL_SOFTDEVICE
 	default 8
 
-config MPSL_SIGNAL_STACK_SIZE
+config MPSL_WORK_STACK_SIZE
```

Comment:
Is there really a good reason for changing this ?
This means that NCS users who have prj.conf files specifying `CONFIG_MPSL_SIGNAL_STACK_SIZE=<value>` will now have to update, but I see no strong reason why this name is so much better.

Reply:
I guess it's because there is now actual work being done in here, not just signalling to the host

Reply:
Because we are introducing mpsl_work concept, and the kconfig should match it.
I think most users would have just left it at the default value, so the impact shouldn't be big.